staging_service/app.py

```diff
@@ -697,4 +698,20 @@ async def app_factory(config: StagingServiceConfig) -> web.Application:
     global auth_client
     auth_client = await KBaseAuth.create(config.auth_url)
 
+    if config.watch_dts_files:
```

Comment:
If this background task is terminated for some reason, nothing will ever turn it back on, is that right?

Reply:
That's right. The service will need to be restarted.

Reply:
I think we need some sort of health check and or way to restart it. 

Here's another case that might happens with some regularity, When the staging service is restarted due to crashing (it happens automatically), and during that time the DTS finishes transferring files.